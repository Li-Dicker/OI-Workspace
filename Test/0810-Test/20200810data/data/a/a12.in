10
197
41636 2110 88972 68091 83347 60968 36565 91067 6130 75850 73046 52509 93362 93132 17732 46210 59736 43720 77186 28659 76531 54755 68418 85571 97105 50716 12220 10470 4588 26338 79760 79401 50625 83624 85683 72166 17987 75128 16851 89700 72596 26477 64722 19606 81296 413 31230 48268 25089 18342 95360 23743 13310 55209 65028 28068 42864 29738 27557 72715 58422 24345 91172 35475 71912 70097 89925 52293 20064 71431 69549 31266 40769 80049 26584 76995 76648 22738 5797 19525 62951 47332 40764 32701 4910 62105 5750 1383 9947 10878 88807 37172 95852 9766 86306 93327 79178 85893 92672 2909 52051 51735 17330 82620 4820 52025 70081 5139 59004 21960 46736 83457 84604 6149 12569 18643 89233 19946 13002 51164 30111 17797 20671 95866 80329 16507 18182 53542 59115 49133 2974 97004 92814 41012 28634 30372 4702 77626 88607 99198 94953 38484 75756 9319 55533 48821 60790 66131 20556 98047 34026 73895 54880 57761 24742 50467 52158 58701 16694 62488 13130 85879 46257 61795 42669 42800 89986 46422 95061 37865 65902 10242 39733 66930 3221 25649 14728 7907 86279 46712 74625 8139 58469 79079 50458 88880 53433 89636 69730 42529 93660 53515 27917 41961 31568 3040 80485 
13160 1792 25231 14240 25587 10127 15709 20775 45963 42173 18556 5424 14379 14050 18957 18783 36659 33012 37310 39836 16435 14346 17590 21165 30064 24211 15941 1720 40421 26885 26987 44847 14897 12806 7632 35887 41651 17159 4353 10512 48249 9605 11881 9210 48311 11584 12520 15661 6644 23242 22777 37085 34560 10359 19590 43597 43612 48873 33121 44157 7425 37721 14103 48950 6554 3641 41722 35358 584 20948 37172 26248 6458 37375 44010 30386 23349 46242 20305 10697 22351 31935 6234 47151 40659 39703 45162 17045 22656 38728 20248 17607 37880 49469 29679 26909 5579 45010 44602 27487 39143 20839 2273 34915 34586 41043 33076 25514 25912 24914 37686 39087 20854 32197 2364 47980 2162 46186 48646 43603 36703 41384 47853 20032 36134 25951 41837 47385 12491 26251 583 20568 16153 48922 7813 45487 29697 20611 45825 43252 327 7306 47598 32937 6484 46833 7224 17543 25479 16040 15756 3292 19399 24773 11657 11782 36489 46502 4555 8994 44739 27377 39189 2897 37302 18655 1202 34196 30622 943 22439 19814 17479 6545 28184 3977 42903 16181 12032 30672 10808 5491 47719 41974 49553 22386 46309 23587 16763 12523 21841 749 5956 25311 963 35888 11417 
196
7513 32878 41152 99810 84724 48706 50356 67617 39941 47743 60148 16571 51082 86371 6841 99524 2784 39951 47124 90123 48310 93869 72842 41434 87401 64649 8949 19221 91772 61352 57495 60332 63449 16583 36501 74668 37399 2319 52395 16815 13580 19486 90707 66804 36669 25372 34488 72099 96811 5346 49029 76089 63795 5105 81460 27049 75750 45033 70540 44217 31164 84976 88890 87968 54683 60314 47494 51675 14897 84299 25163 7307 80759 88947 46761 79449 3001 32212 88103 92354 92201 58873 52241 57207 91546 74677 27141 96306 5997 17141 35701 93912 91651 8457 62113 71535 76239 89913 82339 19257 66812 58681 33134 27885 7829 17151 16285 81496 39918 75027 125 49801 52649 96725 45470 28419 58280 5904 80874 78959 87144 89707 59414 98431 36801 6430 52250 79336 41214 45917 41302 46084 24681 4198 95017 87096 49814 22709 98585 78481 12443 2179 91667 20941 2828 12219 66567 32128 2036 21845 67864 24519 79026 20619 54224 11012 8749 71603 88606 99444 44631 60910 39360 51669 67504 50425 9747 68674 35241 4473 76573 62927 74287 14575 65760 3723 14872 70466 37010 32185 44460 93036 93566 29976 87283 13549 42327 30697 34935 45839 80715 27887 66821 73749 70026 8061 
40048 15831 36795 8997 21939 35542 33072 7044 6247 36366 10524 9914 10381 1080 26448 49764 31894 33110 28247 31932 33248 36704 8490 1294 48218 42372 48545 43908 34484 31853 45763 8259 12182 23594 31894 27157 43726 24951 36803 34907 35749 21761 38640 45752 2495 21661 11686 9991 44891 19801 6550 36551 46354 13206 2093 29501 30573 32395 20842 30582 36193 14980 17451 6042 37754 30099 37253 13705 18290 32033 39836 13921 6780 20406 20708 16980 25999 8394 6302 4391 29570 13220 18354 45645 19453 25381 33768 45633 14758 15381 45156 29062 30977 18105 47989 38568 38780 28910 5226 20105 45587 22076 2113 37627 6532 6901 8726 35420 36718 46560 7459 38279 13463 45210 9109 28605 34544 20564 23782 43928 26242 43163 22808 41196 47406 2038 1050 37014 45182 32545 46078 43028 13123 1047 49418 11674 34918 1007 39178 8430 15568 30762 21648 11042 9955 43101 25659 3151 36067 13350 43800 23132 31168 44128 33048 43643 9209 9294 10406 31115 8628 1799 35629 49470 30156 21820 44809 14070 29126 21091 12362 47834 33909 22815 47450 42114 10692 26319 10336 12140 40487 24541 12392 19551 27054 7689 3733 32016 38725 39965 23917 48934 41811 11611 34144 6452 
197
40003 25325 70419 84428 23031 70394 41557 77645 86197 39887 82496 85347 72724 19493 47000 66823 88636 11098 22747 45398 52889 62146 93405 39118 21725 50639 81661 33841 53770 43681 81657 18748 51007 19358 46360 72684 47962 13199 54200 4483 29787 79351 37440 76907 41472 94873 75533 3114 77950 53451 78648 98073 93046 69424 23742 81529 66741 58744 30337 14959 3019 92405 50071 81642 90128 96430 34790 18336 68794 87474 22487 44735 24298 65093 18629 6675 9866 68073 91973 74327 14169 16104 91829 95260 73625 44899 31071 71395 133 83768 91366 48021 6664 2297 69071 38438 86027 25527 81910 86700 32871 95584 7903 23102 25607 21041 91039 89847 76794 89642 9382 20594 42573 50216 21726 41541 48454 813 76635 72702 46466 92283 79119 54739 73470 33783 59899 39149 93861 62628 83060 83799 97157 80617 16037 77322 4299 58110 39414 25226 55854 93751 89969 3487 55919 51044 68501 56211 20534 27959 17103 31367 22331 65573 15929 9328 25231 1523 24045 86309 47349 47313 58395 6035 74529 13836 68851 25342 50715 69141 35485 88487 14862 4923 77866 25315 59961 64672 58822 38793 10924 62910 13721 44143 21222 6355 47259 35182 36263 29798 17216 59336 27675 32198 1577 95596 59778 
39603 13060 2383 28926 1582 8707 15444 46960 2512 27410 8582 16382 17354 10611 23141 9181 23107 49470 461 48640 29453 45626 35968 45965 6140 34328 40332 11988 48139 4431 45673 3135 10837 43717 22508 40384 8430 43995 49124 34537 1140 8450 46405 44456 7370 39990 35531 39296 47861 38781 26901 10822 30204 25492 33015 9767 11361 17807 6724 19613 48314 18801 47205 3221 9898 20089 23043 18343 29910 25962 15401 47384 3723 8985 14986 18756 393 11037 36208 29408 9931 18577 43408 43404 29569 23694 40600 9299 40010 3787 46981 21834 15807 45454 23245 27537 22664 40679 49496 41981 10184 45886 11884 20198 47173 22645 42267 44986 8849 39808 14511 34179 45895 6987 39999 18654 6873 33901 44606 31652 11093 17665 22501 4700 16863 29419 18645 1789 12940 46359 48818 43894 34089 10828 17554 17302 49773 6869 27926 27124 7 10982 33223 20712 20583 33162 4973 19869 8698 4572 4362 41844 23242 17906 46414 27704 8725 7774 12747 42362 40073 1443 49282 33239 14437 7530 7969 32694 25019 11521 23371 45710 49043 40614 6844 35738 10941 47999 36282 21016 23669 43496 41640 23853 37533 12952 44236 33130 15567 34852 6585 26379 633 45110 32306 3332 37764 
196
33976 67792 3785 69426 95877 50040 50372 51783 58210 16455 60583 62754 1337 56551 77037 34853 36017 51818 34016 13542 86843 71613 63908 85814 51104 20168 37150 55711 28204 73404 52500 2983 27919 13185 19641 87801 94580 47055 18130 40250 7549 258 96630 59003 69025 86609 59326 5650 78308 50627 60808 98067 41228 28982 49379 26418 63472 7602 76853 25338 55693 3836 24060 10659 98860 84735 36394 45556 9606 16235 5480 49316 28968 55176 73414 66630 92560 8341 19203 86240 61930 34365 17506 56257 56688 45424 88678 69796 79853 74453 58945 55691 87523 96829 42262 27059 76185 71318 22898 80912 3482 64900 48861 39004 92112 3589 63872 62400 41925 70992 67801 81947 52175 36251 20667 1760 41782 306 59221 77425 76377 23570 66255 71257 177 40904 57535 91719 69531 81581 67763 87988 61863 52887 40611 18498 76234 89314 31322 6706 1816 31758 62048 30384 71935 68069 34330 36895 85289 97812 63227 68365 7740 5286 35817 7837 88245 6515 74512 26113 42749 37168 41524 72238 9445 53066 86100 15053 16532 10381 14251 78014 4408 39882 47542 69660 58793 97487 66616 42525 59626 48392 14243 78069 53470 56916 48781 48717 92078 82173 45288 71474 5476 10581 65575 24402 
39404 11916 10260 27255 22619 16445 30515 43567 26826 34163 32116 6954 4927 14050 44468 32911 15163 38790 22768 32925 21395 12538 37256 28497 22622 16665 45133 398 1791 48224 41457 12906 2611 32621 24215 48281 35241 22416 42790 47231 5829 35847 29545 18110 24318 46649 23876 39947 42619 20543 48257 30539 44398 18496 29109 41161 24705 33524 20913 26780 39814 41916 14595 36642 462 23498 40566 12044 37701 16954 11660 9299 12690 11134 29230 21014 3829 38322 46805 42522 32509 49673 15828 46880 14001 38493 31745 45813 22818 26010 43131 47137 35050 18440 41634 17435 14766 36895 34931 20163 8698 42761 43771 29348 22673 15128 14357 24575 12172 6574 2304 16432 4974 8872 99 20146 4789 36278 27671 38241 35320 15456 15691 40006 24189 41185 38906 6947 33589 14136 1552 3651 46984 30095 47163 18026 3136 41909 22299 40794 43245 48259 12220 1616 20777 44978 13244 43050 46187 21088 3905 39091 44382 39039 8340 854 8012 19027 2368 25463 17023 1334 11235 4497 26440 24776 31598 12490 35578 46580 31429 301 1643 43298 26324 41106 17289 15851 36393 11720 28310 33206 27093 19314 2546 16673 48450 20636 42141 21588 12470 29456 9320 13282 8222 3995 
200
3147 4977 35888 39037 40399 47293 70530 84357 74035 68110 74598 30601 44635 21658 73000 62506 59070 60486 51702 26649 23358 46822 8773 75714 54795 92822 81368 65269 6141 12895 49494 76764 90985 47823 50223 19153 29630 20179 544 29020 67483 97769 2135 73624 92971 26126 70727 34888 49895 165 73411 24006 15623 9933 10780 22304 88847 41391 88454 84770 81002 7730 83058 63065 10586 96176 77365 29785 38876 26772 74015 93778 95086 48587 69901 89672 40536 73907 17611 39238 14540 79259 1273 55874 13505 26869 42298 98947 92947 19128 55751 60447 34564 76224 14616 1534 95944 69411 62322 4470 59832 96694 19686 20714 18047 24287 69421 99988 63371 10037 97355 7043 59045 56428 88495 69043 54739 66518 17619 93746 33146 39632 84648 21740 56107 74989 37731 66308 797 37765 58062 93121 5644 7028 60031 73364 5540 50042 48107 73291 70765 40531 51219 82806 47028 79000 61774 49330 57273 55365 24524 71695 59091 5169 91244 80014 58058 47621 198 46225 26370 20599 6217 27591 9214 90778 96103 55986 80958 31239 30255 30826 37061 43192 33501 99694 56820 3937 75996 41846 27755 93404 73929 84755 97292 60425 87062 64708 91868 52415 35086 77594 57489 52003 33724 92260 11398 90563 10848 68261 
29901 40931 26942 6136 17844 21821 11540 16472 27734 24717 35011 15913 31715 45699 25923 49994 21416 24747 30280 32530 47325 22437 10536 26638 4987 12237 27672 11745 37500 22 31988 31172 25517 39165 36196 38396 4612 14811 10309 44679 37279 9721 25822 30849 3080 20870 48192 6293 49640 39537 44402 40367 5146 25449 47913 5232 30147 45455 3139 15186 23376 32490 24809 20855 22540 17452 4237 41619 26439 24977 44499 42935 12203 14978 12581 42931 35026 14300 34937 11383 30288 22734 16097 17373 6328 15389 47105 3802 41037 1770 31625 517 3650 30207 24719 3728 24090 11464 43840 18064 47845 13204 25866 4525 44243 16195 49794 963 16571 24122 24114 48911 38947 24545 47838 25804 9854 32121 25382 46311 438 18639 16962 46893 4203 38887 1716 9321 34679 7991 27099 30684 8486 17266 3723 30691 42387 23239 2985 31678 30805 23605 30234 33567 4569 37549 18932 27940 34361 37532 45173 47111 14923 25555 22258 25555 47822 38731 41360 232 37075 37892 49558 17620 13585 15281 25711 2297 34256 19751 42199 24251 41911 38836 3668 42767 1324 35419 22500 20502 29530 12183 31674 8118 34854 22391 40299 9425 30734 9916 49533 37342 18890 6882 30196 29409 27635 26093 38026 12654 
197
93505 38254 5199 36988 20772 54247 39875 71444 23926 3198 10298 78106 58954 94445 44641 24197 74494 89783 33162 29397 99555 41248 20842 59400 28546 62365 31820 16086 96958 5185 21595 65274 53617 64077 9632 59433 41188 98781 69125 87144 26982 20388 98920 24930 89238 48596 21463 75655 72041 71429 62898 49519 76403 86343 71027 77854 30133 92462 24572 48597 21563 7152 55938 16461 5339 40385 16631 86119 52748 81676 65317 89375 22815 15426 74215 53881 27591 50498 30384 81752 11423 12688 17560 26474 86610 77414 18894 53689 96527 75438 12015 48962 80428 99302 11067 98550 85988 17239 43927 93546 43977 18897 59562 61760 54523 8007 56316 42192 97708 23576 18414 94049 34774 1209 91989 79407 59662 73086 43258 75330 73960 39183 55259 74889 5441 47720 69898 33467 9746 28312 60108 41836 19776 83992 80080 26933 9300 73743 17906 90218 41369 76563 74982 44350 69535 47770 62967 9404 8658 17260 75474 78680 57824 258 74695 62632 51624 34869 98969 92285 15614 25347 51817 40257 63612 25692 95807 32371 61331 63400 84195 73493 54561 63968 9269 46389 92289 77847 94247 65016 61390 71925 28821 19397 97714 55468 64199 37130 47505 57907 86252 5369 53921 5867 91020 81700 18212 
41624 23198 28439 46431 48379 10446 33696 40486 38944 47536 26853 42254 23965 40295 34707 19868 12995 42516 23308 40076 19134 40714 13768 4400 6463 1549 6611 8862 34109 712 35043 46423 35766 26194 33165 8099 30551 5076 26152 48804 23250 39936 24605 23342 4916 8384 43379 1855 13719 215 22886 1709 22027 36855 19582 23977 26062 43336 15544 43953 37015 27848 23492 48291 1000 12297 17365 36221 35070 22456 33879 49122 46111 48591 14854 19319 321 6378 13292 39962 33842 47706 8020 23544 29056 20022 47439 21386 24598 1529 5505 20102 46843 23984 14100 12114 8466 10046 16309 31766 37570 43896 27992 7928 12767 31048 21183 13561 38572 12109 1086 23451 39017 12521 47448 19663 21365 27038 1916 5880 43896 18842 24518 37247 17843 49048 4902 29553 43444 18022 16664 36377 24706 10029 27602 39988 29906 10694 12992 33236 18380 30015 16211 29476 32453 9756 41275 23107 33193 22699 29688 35320 43038 33472 8216 2901 4243 44011 4625 31572 47420 36483 31654 28978 9696 39749 16243 41441 13821 43660 34215 47605 10533 18999 24605 47300 19080 42113 2005 23126 36382 25177 47751 36191 26594 10176 24975 9071 30643 1565 33024 47414 29140 39175 7571 21562 41616 
199
72089 92264 93278 80485 36389 51661 66194 19311 66281 75948 70078 69280 22163 92773 39902 61058 12932 10879 78522 98397 64080 65963 64410 64381 75914 1471 25164 67380 22490 31124 53436 9233 65965 43785 50824 54592 96531 61632 84735 2216 20460 5293 53969 91917 91372 91291 71398 56344 31807 15484 77223 63804 29758 98220 93705 52239 61708 54052 27360 47479 74678 61947 13313 91893 71240 43189 27359 84318 16584 91843 60476 84171 57754 49995 15431 49437 30501 96768 94556 26298 65707 36832 12982 24736 30490 28676 89537 9602 49544 19165 68209 29003 14941 12745 81041 80761 82873 76393 82874 35514 34540 58212 63650 21516 65916 23199 33247 93720 99712 88495 98387 18347 85034 37948 90115 51878 81372 22649 94174 64864 95868 49225 1904 37563 47778 65948 55523 91938 15621 57504 26499 18135 40919 63639 74114 88654 11250 18283 2574 60249 75551 86487 89992 60149 79945 37759 57035 99510 76064 86163 31812 33388 73482 14733 69629 15158 54557 53127 52039 97424 97230 10267 50993 99714 39261 58505 65461 10840 4162 50092 72791 97541 47582 30509 60062 28521 50247 68204 47464 39997 65809 48401 82562 48626 43871 37018 89446 54630 5312 86100 2260 51143 70149 7585 60429 91899 18407 49075 24303 
3073 31895 37208 34423 48835 29730 3881 10153 25240 13067 19555 26247 38753 25148 42858 30550 7341 40173 32932 7557 39121 44997 8675 12241 3758 14792 7530 16992 7048 30232 14410 19276 3994 44504 48328 7114 3400 10535 43100 21649 15081 8337 38127 9531 9626 39833 11491 31922 20850 34085 35544 39071 35095 23893 27664 20995 25108 40668 47789 42051 35617 6173 43411 242 13642 29513 15041 45590 5205 46784 37017 41498 25957 24941 25404 43458 6828 34921 32924 40810 37404 26875 2068 26789 42301 31946 26533 44384 23211 5711 21935 43015 13070 38305 25908 15735 9266 19192 49114 47257 33598 10401 48076 23998 48619 26162 14534 40635 9321 41303 40985 4665 22529 1510 3176 34352 45600 38662 20120 43640 42909 37738 14143 38005 31537 42682 34011 46950 44676 28554 37447 25087 30320 35708 46630 34195 14767 37395 6199 7463 42977 42962 47757 33660 43151 45747 7537 30243 39129 2524 48128 36203 47673 36585 38210 37236 34816 27450 38431 49990 45421 14702 46620 29607 21605 11863 19958 12273 4106 13990 8710 1736 31994 30090 33950 8013 47564 25317 34125 3268 4633 47855 25395 12952 27531 8300 40726 895 13618 32081 12973 2932 48077 20930 11837 320 36777 41838 5646 
198
28492 24741 33561 51917 21915 27233 14242 15017 61101 78812 1876 66428 81352 31001 20362 21664 59429 59524 80102 23324 77576 47084 97827 14536 74794 21529 4586 75366 87955 33324 32582 54117 50781 75908 36381 69031 5344 43625 58554 73783 52115 79569 12821 51493 47733 91328 40280 14455 17939 59087 59558 70159 92938 99090 36280 98146 49503 55557 96838 91444 41987 55521 81713 7189 32363 55169 41421 83439 72358 65048 12231 35841 82859 71853 1045 77582 30755 39012 7120 84720 47640 33323 1653 44575 3964 80000 95545 45149 81265 13364 97750 62809 26791 43519 27405 80116 4509 95927 1326 95958 21808 45026 2262 58263 59294 55339 64498 15146 95544 64444 53651 93626 58591 61658 68913 90036 50472 18750 9105 15756 46395 6183 10091 40384 84585 25308 2185 73365 49313 15191 93560 54855 32796 17315 99718 71626 59192 20048 76300 52694 63107 72708 87824 10296 34343 64240 4107 63002 92034 55611 25160 19333 67008 92710 13497 81367 77356 81388 28943 2065 25752 11901 172 95414 71392 66526 34964 96854 54050 1800 99953 95144 34717 85282 75175 90843 81577 80970 10964 74171 99553 48924 11644 85002 90012 20319 14116 3912 16252 8645 46945 96075 81473 30793 65859 98667 85610 24731 
5378 46531 22988 24201 29342 33346 40534 39012 17211 18822 45551 45707 10293 37439 26552 17442 28394 13572 13802 14179 19429 20375 43291 15245 28504 46177 12511 22692 40624 38850 37244 29064 15722 35330 663 3133 12277 20794 7751 20176 33805 27855 16166 38289 25225 41575 33755 15248 12393 26267 48700 27315 30286 46744 15306 18746 32163 30558 12801 24081 20418 16200 9009 18732 10703 23973 33358 16757 18238 30377 41855 34885 4637 22396 35975 20969 34661 44028 12527 16688 3663 3614 44889 2878 49911 21071 9446 8355 14073 43194 13701 30543 22536 8047 8960 19069 6997 49650 30138 37417 49745 25800 45358 45838 46024 7927 26101 43686 43879 47146 903 16867 11975 10364 16617 12493 11498 40530 4084 39866 46061 14007 35333 10273 10883 1901 47875 41740 1235 27806 9064 43666 17224 11168 45631 42291 40119 37116 44945 28447 25411 26719 8662 9497 37775 45429 31556 29225 12057 46084 28989 40814 37562 15015 46425 23229 49338 14948 11394 15617 18026 30098 5738 45432 46663 20538 45189 12107 29820 15442 11930 17584 26602 30280 11421 33499 23599 3206 7068 13698 12737 28720 21833 13876 47986 5792 39145 4840 4922 14349 22716 27204 2302 37508 13386 36802 29556 44177 
196
98747 17488 66291 78785 94268 1109 98822 40074 29026 28370 86221 40984 83311 63572 39560 14491 71940 89205 97129 47387 25465 31621 35576 64890 98554 53470 70506 12355 48482 59853 82570 30458 98756 14871 95151 92117 98067 7297 79251 32127 97163 66875 27558 54415 35272 3716 12105 38825 45407 77394 61510 87637 71649 93204 56157 6056 43092 25675 58245 94803 11171 92008 75661 73227 59496 42523 95111 9484 82091 75064 97127 50170 77161 43791 92307 60363 18321 37169 80654 33755 64940 34494 94337 40874 15607 64333 91239 33924 82058 85180 60610 49457 49136 6768 68431 26463 39342 65434 81351 98691 1946 51921 94266 99972 85715 11817 81687 40379 8539 92407 7495 28904 489 35357 84536 45288 52248 62891 92516 98608 66369 30687 83638 40038 12204 42490 65832 26290 49226 37246 18227 7796 93489 67309 99469 81032 83633 12316 47761 31914 62271 91634 38562 70272 66106 9069 73573 57121 92835 60960 60839 69956 7899 75614 90950 3755 99366 93428 87508 92453 80291 90184 89849 99547 72815 63679 83960 28508 54824 68582 57285 50969 86698 94414 68911 21944 64507 97717 15909 50689 97811 81480 77310 27472 88653 85911 91962 74924 93215 96254 38427 97218 9002 45122 91641 27063 
5260 32260 4904 21337 48151 6387 21766 24337 48380 11343 10330 12462 12228 8267 28811 23311 5306 14661 7206 13081 47075 36147 29039 43901 17819 26558 21400 30408 26191 13066 15011 49446 8671 9761 36996 48385 35385 24873 18514 45128 38830 48788 49007 11683 42151 5461 38201 20332 26097 49345 41066 25113 42518 255 42622 798 16469 42747 42892 43964 7298 44402 24078 286 38309 49709 34520 6687 16189 16617 42400 8884 17847 26596 36587 41862 16762 9511 22836 7096 9846 19875 42787 21278 19285 42727 25772 3871 28353 28174 8465 3061 37066 24235 17038 18201 42135 47149 44069 9548 12621 30443 49994 40619 18892 17714 25508 17667 23319 25087 7556 3463 30046 25394 26891 22092 23598 44427 45388 33641 33139 42363 29187 15136 20680 45516 16637 40980 13268 1920 32556 36113 38551 38612 5774 11919 37865 13711 46465 41045 471 20997 12737 10617 48075 19551 36155 40165 21724 31432 22878 4790 24308 46513 33331 6576 35069 44984 36018 43967 49670 2836 31037 32455 502 49247 17733 35379 37874 15483 37779 18731 10312 5026 799 9135 21890 17024 49212 6730 20882 4958 3095 22678 11180 3265 22664 41047 33722 48354 46227 14953 32041 5970 9544 35321 
196
98470 64397 75907 47364 2999 26675 10288 22357 40363 29476 71869 25230 72438 87274 87812 21081 63944 1072 71346 57989 13415 31485 93066 76579 6774 89377 75642 30435 99397 91109 61516 76181 79585 54001 25913 44924 71347 89161 82118 63204 24596 5158 64860 94824 67368 15779 14396 17895 73831 48510 5898 33705 56714 24358 76185 22513 93923 8989 77228 19426 25766 73013 95171 78412 9994 15170 51508 7633 239 92991 75647 55932 96717 93696 89917 95843 60577 749 33214 52722 31576 48592 11327 61528 98590 88998 64900 43615 13672 22901 26287 59938 39639 45533 62904 9525 16953 14161 78452 44592 66461 72021 67826 49514 48555 62077 80256 41486 67378 14584 94229 6341 31428 39623 23269 55374 89556 70816 83883 90300 498 9662 95307 76723 66220 87584 85048 24035 42412 31938 50559 45127 68230 45393 88144 33649 95073 98729 52706 94094 53957 44396 63793 15434 97505 83057 21443 83363 32651 28239 65741 67198 80937 95299 42090 4173 43731 99518 72299 61758 79178 33075 29735 52788 44485 46964 13551 90080 33174 72321 68211 35776 4542 31499 59102 55724 60198 49964 48225 62790 50663 70552 67915 57510 55014 96185 37444 32595 30147 58569 82799 47816 36418 38043 58098 94747 
8031 25307 32227 8645 16101 37916 16906 33790 3148 36335 23941 44560 21297 11419 34672 39924 38902 17455 10752 43033 3538 3022 49525 41156 18048 13638 23553 18214 31185 1316 8788 6137 1508 1606 30987 3935 31461 6646 10381 20771 14491 9799 40198 48520 46590 35919 43076 9264 28978 10151 45408 12739 41057 21940 17563 4472 36471 34946 33433 35299 19873 16143 15789 1997 49637 34074 17535 33588 25178 19654 15731 21995 34180 18522 35945 10134 29615 2923 23830 39090 24548 43026 43317 14081 49834 47788 35375 39320 12317 15851 16575 2178 19899 37506 35812 9076 33391 36974 9393 38394 15463 25351 33229 49979 34896 10873 11110 33672 33407 19783 23454 3973 18281 23110 25814 19618 38059 48604 25650 21331 28316 49240 36350 35144 18741 33080 38119 5764 7185 38663 31991 27772 36367 23820 14542 45170 18994 49036 45299 37817 33772 33154 33950 236 5989 24971 22142 16326 46383 45520 33149 16814 14382 15221 27874 47067 33272 4706 12294 19055 40945 20640 41849 12861 16817 7238 19244 38737 23321 34495 41553 12722 40629 12805 34881 8688 16225 48037 42918 45695 28507 25225 29192 38985 12546 20796 46588 5655 24435 23179 26180 6990 24969 47146 14398 6198 
