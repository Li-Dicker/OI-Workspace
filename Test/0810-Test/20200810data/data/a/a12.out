Case #1: 14077501
Case #2: 14424453
Case #3: 13856790
Case #4: 14039524
Case #5: 14250311
Case #6: 14606344
Case #7: 15488077
Case #8: 14508842
Case #9: 15856336
Case #10: 15017392
